app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -19,29 +19,29 @@
 /**
  * Simple login activity class.
  */
-public final class LoginActivity extends Activity {
+public final class LoginActivity extends AppCompatActivity {
```

Comment:
I think it's AppCompatActivity, because we have an action bar at the top (although it is one with no actions?!?) and AppCompatActivity is the recommended activity for modern Android applications using an action bar